common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -1,5 +1,5 @@
 $sequence--border-color: #C8C8C8;
-
+$link-color: rgb(26, 161, 222);
```

Comment:
instead of using this we should use one of our base blues from _variables.scss, even if it is $m-blue;